firmware/controllers/algo/rusefi_enums.h

```diff
@@ -670,42 +670,25 @@ typedef enum {
 } debug_mode_e;
 
 typedef enum {
-	MT_CUSTOM = 0,
-	MT_DENSO183 = 1,
-	/**
-	 * 20 to 250 kPa (2.9 to 36.3 psi) 0.2 to 4.9 V OUTPUT
```

Comment:
why reformat and remove comments?

Reply:
Reordering done for a tidy list in drop-down.

Comments not really useful to anyone in this file..

Would fit better in help-file, TS or documentation